### ISSUE TITLE
Rotation 'locks' at certain angles

### ISSUE THREAD

**Issue by prefrontalvortex** [maartenbreddels/ipyvolume#7]
I've noticed I can't freely rotate continuously in every axis. Trying to rotate around the pole that corresponds to numpy `axis=2`  only goes 180 degrees lock-to-lock. It feels like it is a limitation of the underlying 3D engine. Any idea if there is a way to work around this?

**Comment by maartenbreddels** [comment]
Yes, quite annoying, it has indeed to do with threejs, probably configurable. Thx for the report, I'll look into it soon, i sometimes transpose the numpy array to work around it.

**Comment by maartenbreddels** [comment]
This is fixed by ba66069f43b1938bc4f6eb8fe765e68b7c826e8d , the default is the more 'natural' trackball navigation, the old method is still available:
```python
fig.camera_control = 'orbit' # stops at poles
fig.camera_control = 'trackball' # trackball behavious (default now)
```

**Comment by Tronic** [comment]
The orbit mode would be highly preferred, if it worked correctly, that is, sideways movement always adjusts rotation ψ about the Y axis, while vertical movement adjusts pitch angle φ. This is how all professional 3d modeling software I know of implements rotation, and for a good reason. This is also how ipyvolume orbit rotation works during the very first rotation, until mouse button is released.

Both modes as currently implemented very quickly result in rotations that are awkward and quite broken. In particular, returning to any prior angle or reaching any straight angle is extremely difficult.
